kuberay-operator.yaml

```diff
@@ -30,6 +30,16 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: kuberay-operator-compat
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          # The helm chart expects the kuberay-operator binaries to be in / instead of /usr/bin
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+      - uses: strip
```

Comment:
strip is not necessary here as we are not building a sub package and manager already uses strip on line 31

```suggestion
```